.circleci/config.yml

```diff
@@ -20,6 +20,14 @@ commands:
       - run:
           name: Checkout Gutenberg
           command: git submodule update --init --recursive
+  add-jest-reporter:
+    steps:
+      - run:
+          name: Install JUnit coverage reporter
+          command: yarn add --dev jest-junit
```

Comment:
I think that will work on CI but it means that when running locally, the `jest-junit` package won't be found since we won't have it installed. Any particular reason to only install it on CI?

In any case, it'd be good if we can run `./bin/ci-checks-js.sh` locally too, both to verify it and to have an easy way to start all the tests. WDYT?

Reply:
> Any particular reason to only install it on CI?

I didn't think the reports would be useful locally but it doesn't hurt. As a fix I included it in the package.json and ignore the output in the gitignore.